include/reloader.h

```diff
@@ -5,12 +5,12 @@
 #include <vector>
 
 #include "configcontainer.h"
+#include "curlhandle.h"
```

Comment:
I think a forward declaration is a better approach here, because:

1) `Reloader` doesn't have `CurlHandle` members, so it doesn't *need* the definition of `CurlHandle` — only its declaration;
2) `#include`s in headers are "viral" in a sense that anyone who includes this header also transitively includes curlhandle.h, even if they don't need it. This can cause unnecessary recompilation when curlhandle.h is changed.

So I'd prefer to use forward declarations as much as possible, and put `#include`s into cpp files instead whenever possible.

Same goes for rssparser.h, below.

Reply:
I will change it in reloader.h.

> Same goes for rssparser.h, below.

It looks like `rssparser.h` uses `CurlHandle` as a member variable.

Reply:
`RssParser` only contains a pointer, so it doesn't need the class definition either: all pointers are the same size, and the compiler only needs the definition to figure out the size of the object.

Reply:
Good point, changes to rssparser.h have been undone.